src/model/storage.py

```diff
@@ -42,6 +46,7 @@ def __init__(self, model, organism_id, biomass_reaction):
         self.model = model
         # Use the cplex solver for performance
         self.model.solver = 'cplex'
+        self.project_id = project_id
         self.organism_id = organism_id
         self.biomass_reaction = biomass_reaction
```

Comment:
maybe we have to add the preferred_map_id here?

Reply:
Thanks :slightly_smiling_face: but I think that since we are only doing simulations here, the map visualization details aren't really relevant. It will still be available for other services querying the model storage.